### DISCUSSION TITLE
[71340]索尼克世代[本体]

### DISCUSSION THREAD

**Discussion by YuuKi-Z** [pjy612/SteamManifestCache#182] [Share]
`Appid:71340`
keys:
```
71341;e9a94a9cea6b1e07fb4ab98e4186c7af965a32ace868d3ed5bf8151adc2016e0
```
![image](https://github.com/pjy612/SteamManifestCache/assets/66229359/7c0f914b-be43-4ef8-99f5-0a7b50b6bd84)
[71341_190831558827569965.zip](https://github.com/pjy612/SteamManifestCache/files/13777067/71341_190831558827569965.zip)

email:[2596666711@qq.com](mailto:2596666711@qq.com)
code:FCB7DA7987C6C6B7

**Comment by pjy612** [comment]
确定清单文件没传错？

**Reply by YuuKi-Z** [reply]
确实传错了，已改正
